src/main/drivers/pwm_output.c

```diff
@@ -328,12 +315,21 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
         IOConfigGPIO(motors[motorIndex].io, IOCFG_AF_PP);
 #endif
 
+        /* standard PWM outputs */
         if (useUnsyncedPwm) {
-            const uint32_t hz = timerMhzCounter * 1000000;
-            pwmOutConfig(&motors[motorIndex], timerHardware, timerMhzCounter, hz / motorConfig->motorPwmRate, idlePulse, motorConfig->motorPwmInversion);
-        } else {
-            pwmOutConfig(&motors[motorIndex], timerHardware, timerMhzCounter, 0xFFFF, 0, motorConfig->motorPwmInversion);
+            pwmRate = motorConfig->motorPwmRate;
         }
+        const uint32_t clock = timerClock(timerHardware->tim);
+        const uint16_t prescaler = ceilf((clock / pwmRate) / 65535.0f); /* used to find the desired timer frequency for max resolution */
```

Comment:
65535 is 0xFFFF-1

But I believe it is clearer to the reader of the intention to see the ceilf. This is not a time critical piece of code, only run once (init), so I am opting for clarity over optimisation.